src/tests/ftest/util/job_manager_utils.py

```diff
@@ -1116,3 +1116,98 @@ def dump_logs(self, hosts=None, timestamp=None):
             if hosts is None:
                 hosts = self._hosts
             self.display_log_data(self.get_log_data(hosts, timestamp))
+
+
+class Clush(JobManager):
+    # pylint: disable=too-many-public-methods
+    """A class for the clush job manager command."""
+
+    def __init__(self, job):
+        """Create a Clush object.
+
+        Args:
+            job (ExecutableCommand): command object to manage.
+        """
+        super().__init__("/run/clush/*", "clush", job)
+        self.verbose = True
+        self.timeout = 60
+
+    def __str__(self):
+        """Return the command with all of its defined parameters as a string.
+
+        Returns:
+            str: the command with all the defined parameters
+
+        """
+        commands = [super().__str__(), "-w {}".format(self.hosts), str(self.job)]
+        return " ".join(commands)
+
+    def assign_hosts(self, hosts, path=None, slots=None):
+        """Assign the hosts to use with the command (--hostfile).
+
+        Args:
+            hosts (NodeSet): hosts to specify in the hostfile
+            path (str, optional): not used. Defaults to None.
+            slots (int, optional): not used. Defaults to None.
```

Comment:
Not used? Is this because of inheritance?

Reply:
Yes.